contracts/EntryPoint.sol

```diff
@@ -208,7 +323,7 @@ contract EntryPoint is StakeManager {
     }
 
     // create the sender's contract if needed.
-    function _createSenderIfNeeded(UserOperation calldata op) internal {
+    function _createSenderIfNeeded(UserOperation calldata op) internal returns (bool){
```

Comment:
please DM me with the attack vector.
AFAIK, a proper constructor for a wallet (which contains full signer infromation - e.g. public key, aggregator and entrypoint addresses) protects against such attack.
the attacker can "front-run" the wallet owner and create the wallet (with the real owner's public key) - but only the real owner can use it, since it is only callable via the entrypoint, after  signature validation (validateUserOp validates the aggregator address, and entrypoint calls validateSignatures, which validates this userOp's hash and signature are valid)